sky_alert/endpoint.py

```diff
@@ -7,6 +14,27 @@
 def healthz():
     return 'OK'
 
-@app.get('/sunrise')
-def sunrise():
-    return '06:00'
\ No newline at end of file
+@app.get('/sundata')
+def sun_data(lat=0, lon=0):
+
+    headers = {
+        'Content-Type': 'application/json',
+    }
+
+    params = {
+        "lat": lat,
+        "lon": lon,
+        "appid": os.environ["OPENWEATHER_API_KEY"]
+    }
+    
+    res = requests.get(os.environ["OPENWEATHER_API_URL"], params=params, headers=headers)
+
+    sunrise = datetime.datetime.utcfromtimestamp(res.json()['current']['sunrise'])
+    sunset = datetime.datetime.utcfromtimestamp(res.json()['current']['sunset'])
```

Comment:
what would happen if openweather's API response changes and / or if `current` isn't in the response? What should we return?

Reply:
Good question, I'm refactoring the code to return None if the structure of the response changes - let me know if you have any advice on what should be returned.